requirements.txt

```diff
@@ -0,0 +1,20 @@
+asgiref==3.5.2
```

Comment:
these requirements are more than what's expected in the first task. Expecting packages related to django and postgres only